bigquery_etl/util/temp_table.py

```diff
@@ -0,0 +1,43 @@
+"""Generate temporary tables."""
```

Comment:
It's really nice to have some of these items that existed in previous scripts factored out into more maintainable and better-documented modules.